### PR TITLE
C24_WMDE_Desktop_DE_08

### PR DESCRIPTION
**Pull request by moiikana**
- based on de 07
- tests features like softclose again

- [x] alreadydonated footer that ignores for 4 weeks
- [x] progress bar
- - - 
- [x] ctrl has no softclose but also sets item in local storage 
- [x] var has softclose and sets item in local storage
- [x] 	Submitting the banner form logs an additional unique event,
       	- if the user closed a banner, clicked "Not now" in the soft close or ignored the soft close before.
    		- if the user clicked "Maybe later" in the soft close.

https://phabricator.wikimedia.org/T375606

### CODE REVIEW

File: banners/desktop/C24_WMDE_Desktop_DE_08/banner_ctrl.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const app = createVueApp( BannerConductor, {
+	page,
+	bannerConfig: {
+		delay: runtimeEnvironment.getBannerDelay( 7500 ),
+		transitionDuration: 1000
+	},
+	bannerProps: {
+		useOfFundsContent: localeFactory.getUseOfFundsLoader().getContent(),
+		remainingImpressions: impressionCount.getRemainingImpressions( page.getMaxBannerImpressions( 'desktop' ) ),
+		donationLink: createFallbackDonationURL( page.getTracking(), impressionCount )
```
Comment (Abban) [inline banners/desktop/C24_WMDE_Desktop_DE_08/banner_ctrl.ts:50]:
```suggestion
		donationLink: createFallbackDonationURL( page.getTracking(), impressionCount ),
		localCloseTracker: new LocalStorageCloseTracker()
```

**Review by Abban — APPROVED** [review]
Spotted a couple of things:

You need to add the extra close choice handler to PageWPORG like I did in the [mobile banner](https://github.com/wmde/fundraising-banners/pull/592/files#diff-8b3aa8699d4b1b0db275251088bf6b7b2c00d6dc9961f8cf458f74433f0a8663R128). (Maybe this PR needs a rebase?)

File: banners/desktop/C24_WMDE_Desktop_DE_08/banner_ctrl.ts (hunk 1 above)
Comment (moiikana) [reply banners/desktop/C24_WMDE_Desktop_DE_08/banner_ctrl.ts:50]: sorry, didn't have my last changes pushed. 
the changes are pushed now.

**Comment by moiikana** [comment]
rebased because https://github.com/wmde/fundraising-banners/pull/597 was merged

**Comment by moiikana** [comment]
rebased main

**Comment by gbirke** [comment]
In case you're rebasing - there is a typo in your 2nd commit title "Peprare" -> "Prepare"

File: banners/desktop/C24_WMDE_Desktop_DE_08/messages_var.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	...FallbackBanner,
+	'already-donated-go-away-button': 'Im Moment nicht',
+	'soft-close-prompt': 'Dürfen wir später nochmal fragen?',
+	'upgrade-to-yearly-copy': `<p>Jedes Jahr sind wir auf Menschen wie Sie angewiesen. Jährliche Spenden helfen uns besonders und ermöglichen langfristige Weiterentwicklungen.</p>
+		<p>Sie gehen kein Risiko ein: Jederzeit formlos zu sofort kündbar.</p>`,
+	'upgrade-to-yearly-no': 'Nein, ich spende einmalig {{amount}}',
+	'upgrade-to-yearly-yes': 'Ja, ich spende {{amount}} jährlich',
+	'campaign-day-only-n-days': 'Heute sind es nur noch {{days}} Tage bis zum Ende unserer Spendenkampagne.',
+	'double-progress-close': 'Das wird knapp.',
+	'missing-amount': 'Uns fehlen noch'
```
Comment (gbirke) [inline banners/desktop/C24_WMDE_Desktop_DE_08/messages_var.ts:30]: You could remove these, it's a leftover from the 2023 campaign (and the double progress bar feature) and not used in the content.

**Comment by moiikana** [comment]
rebased main,
resolved merge conflicts,
fixed typo in "prepare" commit

**Review by gbirke — APPROVED** [review]

File: banners/desktop/C24_WMDE_Desktop_DE_08/messages_var.ts (hunk 2 above)
Comment (moiikana) [reply banners/desktop/C24_WMDE_Desktop_DE_08/messages_var.ts:30]: we will use the Rennbanner/double progress bar again in desktopDE11 next Friday, so maybe we can leave it in for now

**Pull request merged**